### PR TITLE
Target calories and BMI instant update & bug resolution with update button

### LINKED ISSUE #32: BMI calculation needs to be automatically reflected when user updates height or weight.
Updating profile information such as age, weight and height does not automatically reload BMI calculation. Currently, the changes are seen only on reloading the page.

### PR DESCRIPTION
**Pull request by srina1h**
Closes #32

### CODE REVIEW

File: backend/base.py [hunk 1]
```diff
@@ -530,6 +530,11 @@ def goalsUpdate(): # pragma: no cover
         "email": current_user,
     }
     try:
+        current_user = get_jwt_identity()
```
Comment (shashank-madan) [inline backend/base.py:533]: Does current_user need to be assigned inside the try block again? It seems to have already been done outside.

File: backend/base.py [hunk 2]
```diff
@@ -530,6 +530,11 @@ def goalsUpdate(): # pragma: no cover
...
+        profile = mongo.user.find_one({"email": current_user})
```
Comment (shashank-madan) [inline backend/base.py:534]: Is it possible to use the "query" variable which is set to the same as what is being queried?

**Review by shashank-madan — CHANGES_REQUESTED** [review]

File: backend/base.py (hunk 2 above)
Comment (srina1h) [reply backend/base.py]: smart

File: backend/base.py (hunk 1 above)
Comment (srina1h) [reply backend/base.py]: oops

**Review by shashank-madan — APPROVED** [review]

**Pull request merged**